src/Interfaces/AbstractZonotope.jl

```diff
@@ -884,6 +885,31 @@ function _remove_redundant_generators_1d(G::AbstractMatrix)
     return hcat(g)
 end
 
+"""
+    remove_small_generators(Z::AbstractZonotope, tol::Real=1e-4; p::Real=Inf)
```

Comment:
```suggestion
    remove_small_generators(Z::AbstractZonotope, [tol]::Real=1e-4, [p]::Real=Inf)
```
At least that is the current signature.